deploy/kubernetes/statefulset.yaml

```diff
@@ -11,6 +11,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: csi-provisioner
+  namespace: kube-system
```

Comment:
It would be great to see these in a namespace other than kube-system. External things would ideally be placed in a separate namespace

Reply:
done